web5-spec-test-vectors.go

```diff
@@ -0,0 +1,36 @@
+package web5go
```

Comment:
thoughts on `package web5`? i feel like the `go` is implied

Reply:
nice dude yeah agreed, done